paystack/src/main/java/co/paystack/android/api/PaystackRepositoryImpl.kt

```diff
@@ -71,6 +71,14 @@ internal class PaystackRepositoryImpl(private val apiService: PaystackApiService
         )
     }
 
+    override fun getTransactionWithAccessCode(accessCode: String, callback: ApiCallback<TransactionInitResponse>) {
```

Comment:
Can we write tests for this repo method please since its new code and in Kotlin. Should be east to test.

Reply:
Done @Peter-John-paystack